Doc/library/re.rst

```diff
@@ -33,7 +33,8 @@ usage of the backslash in string literals now generate a :exc:`SyntaxWarning`
 and in the future this will become a :exc:`SyntaxError`. This behaviour
 will happen even if it is a valid escape sequence for a regular expression.
 
-The solution is to use Python's raw string notation for regular expression
+The solution is to use Python's :ref:`raw string notation <raw-string-notation>`
+for regular expression
```

Comment:
> This is written as if it links to general raw strings in Python (e.g. under Lexical Analysis), rather than being a forward reference to an RE-specific section.

Do you mean something like this?

```
The solution is to use Python's :ref:`raw string notation  for regular expression patterns <raw-string-notation>`
```

> I don’t find linking to a subheading from the middle of the introduction section is helpful; why not use the table of contents?

Does the reader know the term is in the table of contents? I guess don't.

Reply:
If we do need the forward reference here, I guess this longer link text is less surprising. I would include “Python’s” in the link text as well though.

Reply:
Done